frontend/src/common/types/component/component.ts

```diff
@@ -0,0 +1 @@
+export type ComponentType<P> = (props: P) => JSX.Element;
```

Comment:
**This is redundant!**
React already has the `FC` type

Reply:
I fixed that